modules/compute/container-registry/README.md

```diff
@@ -0,0 +1,91 @@
+| `diagnosticStorageAccountId`            | `string` | No       | Resource ID of the diagnostic storage account.                                                                                                                                                                                                                                                                                                                                                       |
+| `diagnosticWorkspaceId`                 | `string` | No       | Resource ID of the diagnostic log analytics workspace.                                                                                                                                                                                                                                                                                                                                               |
+| `diagnosticEventHubAuthorizationRuleId` | `string` | No       | Resource ID of the diagnostic event hub authorization rule for the Event Hubs namespace in which the event hub should be created or streamed to.                                                                                                                                                                                                                                                     |
+| `diagnosticEventHubName`                | `string` | No       | Name of the diagnostic event hub within the namespace to which logs are streamed. Without this, an event hub is created for each log category.                                                                                                                                                                                                                                                       |
+| `logsToEnable`                          | `array`  | No       | The name of logs that will be streamed.                                                                                                                                                                                                                                                                                                                                                              |
+| `metricsToEnable`                       | `array`  | No       | The name of metrics that will be streamed.                                                                                                                                                                                                                                                                                                                                                           |
+
+## Outputs
```

Comment:
Can you add a new output for LoginServer? I've found that to be pretty handy in other modules where i've used ACR.

Reply:
Good shout, I have added that, thanks!